src/components/pages/AdminHome/AdminSidebar.js

```diff
@@ -35,45 +29,23 @@ function AdminSidebar() {
       onCollapse={onCollapse}
     >
       <div className="logo" />
-      <Menu theme="light" defaultSelectedKeys={['1']} mode="inline">
-        <Menu.Item
-          key="1"
-          icon={<ThunderboltOutlined />}
-          className="dashboard-logo"
-        >
-          Coderheroes
-        </Menu.Item>
-
-        <Menu.Item key="2" icon={<HomeOutlined />}>
-          <a href="/admin">Home</a>
+      <Menu theme="light" defaultSelectedKeys={[path]} mode="inline">
```

Comment:
right now it's light, but will this need to be dynamic? 

the reason I'm wondering is do we need to be explicit? Or is light mode the default?